frontend/ui/impl/src/test/resources/expected-error-report-uris/for_multiple_events.txt

```diff
@@ -0,0 +1,52 @@
+https://github.com/VirtusLab/git-machete-intellij-plugin/issues/new
```

Comment:
Split into multiple lines... for the sake of easier diffing; ofc lines get joined for comparison